src/core/Form/Checkbox/Checkbox.md

```diff
@@ -52,3 +52,29 @@ const [checked, setChecked] = React.useState(false);
   </Checkbox>
 </>;
 ```
+
+### Checkboxes in group
+
+```js
+import { Checkbox, CheckboxGroup } from 'suomifi-ui-components';
+import React from 'react';
+
+<>
+  <CheckboxGroup
+    labelText="Checkboxes in group"
+    groupHintText="Example hint text"
+  >
+    <Checkbox defaultChecked hintText="Example hint text">
+      Regular checkbox that is checked and has a hint text
+    </Checkbox>
+
+    <Checkbox hintText="Example hint text">
+      Regular checkbox with a hint text
+    </Checkbox>
+
+    <Checkbox hintText="Example hint text">
+      Regular checkbox that has a hint text
```

Comment:
I'm slightly bothered that this is identical with the previous one bat the label/content is different 😅 could the last one be without hint text to show that's possible as well?

Reply:
The latter example is now changed to be just "_Regular checkbox_"